### ISSUE TITLE
[SF][A11Y][ui5-input] The placeholder is not announced by JAWS

### ISSUE THREAD

**Issue by BennyHuang** [SAP/ui5-webcomponents#4688]
### **Bug Description**
The placeholder for the input field is not announced by JAWS. But it is announced by NVDA and VoiceOver.

### **Expected Behavior**
The placeholder text in input should be announced by screen reader JAWS.

### **Context**
- UI5 Web Components version: 1.0.2
- OS/Platform: Windows
- Browser: Chrome
- Affected component: ui5-input

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SAP Successfactors

**Comment by Shtilianova** [comment]
Hello @BennyHuang ,
The issue is not reproducible in our example [https://sap.github.io/ui5-webcomponents/playground/components/Input/](https://sap.github.io/ui5-webcomponents/playground/components/Input/). 
Please provide a way to reproduce it.
Regards,
Diana

**Comment by gmkv** [comment]
Closing issue due to inactivity.
